integration/simulation/validate_chain.go

```diff
@@ -248,6 +249,10 @@ func checkBlockchainOfObscuroNode(t *testing.T, nodeClient *obscuroclient.Client
 	for _, wallet := range s.TxInjector.wallets {
 		total += balance(nodeClient, wallet.Address)
 	}
+
+	// also check for the worker wallet address
+	total += balance(nodeClient, s.TxInjector.ethWallet.Address())
```

Comment:
does this wallet ever have a balance >0?

Reply:
Cleaned this up like discussed offline